packages/mask/src/plugins/Collectible/hooks/useAssetOrder.ts

```diff
@@ -16,13 +16,12 @@ export function useAssetOrder(provider: NonFungibleAssetProvider, token?: Collec
                 const openSeaResponse = await PluginCollectibleRPC.getAssetFromSDK(token.contractAddress, token.tokenId)
                 const getPrice = (order: Order) =>
                     getOrderUnitPrice(
-                        order.currentPrice?.toFixed(),
+                        order.currentPrice as unknown as string,
```

Comment:
As I can see from the code, the original type is BigNumber. Does the latest version of opensae-js fix this issue?

![image](https://user-images.githubusercontent.com/52657989/147627325-96bf2858-da60-4772-9da0-3206ee6dc9f2.png)

Reply:
Up to opensea-js latest version 1.2.7, type still BigNumber when the response type is string, assert type as AssetOrder and its works fine